lib/middleware/request-handler.ts

```diff
@@ -406,11 +412,17 @@ export function elevateRequest (req: Interface.IOurRequest,
         res.write(text + JSON.stringify(data));
     };
 
-    res.sendEnd = (status: string, message: string): void => {
-        var text = util.format('%s%s,%s}',
+    res.sendEnd = (status: string,
```

Comment:
I would recommend we have a `sendRaw` that does no wrapping/structure of the response and just sends the entire buffer-in full to the client.  Then you can just send the correlation ids that way.